app/views/layouts/application.html.erb

```diff
@@ -50,6 +50,9 @@
         <%= yield %>
       </main>
     </section>
-
+    <% if @logged_in_user %>
+      <h3>Hit count: <%= @logged_in_user.hit_count %>
```

Comment:
Here is what all I want to see in the hit counter
Current session's hit count
Current session's current page hit count
Overall user's hit count
Overall user's current page hit count
Overall hit count
Overall page hit count

Reply:
Done with - 

- Current session's hit count
- Overall user's hit count
- Overall hit count

Need to discuss about the rest